task-requests/README.md

```diff
@@ -181,17 +182,20 @@ Content: {
 		 }
 ```
     
-## **PATCH /task-requests/:id**
+## **PATCH /taskRequests/:**
 
 - **Params**  
-  _Required:_ `id=[string]`
 - **Query**  
   - Optional: `dev=[boolean]` (`dev` is passed to get all tasks requests in the developer mode with features that are flagged)
+  - Required: `action=[approve | reject]` (`action` action either approves or rejects the task request of the given id)
 - **Headers**  
   Content-Type: application/json
 - **Body** 
 ```
-	content: { <task_request_object> }
+	content: {
+			taskRequestId : <id>
+			userId : <userId>
+			}
```

Comment:
can you check the formatting . parentheses seems out of place.

Reply:
Done.